### DISCUSSION TITLE
Where can I get the config files for compiling

### DISCUSSION THREAD

**Discussion by ltrsunny** [linux-chenxing/linux-chenxing.org#42] [General]
I want to compile u-boot & kernel for SSD202, but I can not find any related 'defconfig' files in current branches. 
(They look just the forks from official ones)

**Comment by fifteenhex** [comment]
This is a bit roundabout but the settings used at the moment are covered in the buildroot config used for a ssd202d device:
https://github.com/fifteenhex/buildroot_idosom2d01/blob/master/br2idosom2d01/configs/idosom2d01_defconfig

**Reply by ltrsunny** [reply]
Ok thanks, let me try.
